nav-app/src/app/tabs/tabs.component.html

```diff
@@ -251,6 +251,36 @@ <h3>WARNING</h3>
                         </td>
                     </tr>
 
+                    <!-- Links -->
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                <mat-select placeholder="links" [(ngModel)]="links" [disabled]="!domain">
+                                    <mat-option [value]="null">none</mat-option>
+                                    <mat-option *ngFor="let vm of getLayers()" [value]=vm>{{vm.name}}</mat-option>
+                                </mat-select>
+                            </mat-form-field>
+                        </td>
+                        <td [ngClass]="{disabled: !domain}">
+                            Choose which layer to import links from. Leave blank to initialize without links.
+                        </td>
+                    </tr>
+
+                    <!-- Metadata -->
+                    <tr>
+                        <td>
+                            <mat-form-field>
+                                <mat-select placeholder="metadata" [(ngModel)]="metadata" [disabled]="!domain">
+                                    <mat-option [value]="null">none</mat-option>
+                                    <mat-option *ngFor="let vm of getLayers()" [value]=vm>{{vm.name}}</mat-option>
+                                </mat-select>
+                            </mat-form-field>
+                        </td>
+                        <td [ngClass]="{disabled: !domain}">
+                            Choose which layer to import technique metadata from. Leave blank to initialize without metadata.
```

Comment:
There's no way to import layer-level metadata, only technique level metadata. We may want to widen the scope of this option to include layer-level metadata.